presto-docs/src/main/sphinx/release/teradata-additions.rst

```diff
@@ -0,0 +1,24 @@
+==================
+Teradata Additions
+==================
+
+The following are additions that Teradata has added to Facebook's 0.148 release of presto
+
```

Comment:
Neutrino additions:
Add support for FLOAT in the Hive connector, as well as FLOAT-specific functions.